src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -45,7 +45,8 @@ object ScalafixPlugin extends AutoPlugin {
 
     val scalafixOnCompile: SettingKey[Boolean] =
       settingKey[Boolean](
-        "Run Scalafix rule(s) declared in .scalafix.conf on compilation and fail on lint errors. " +
+        "Run Scalafix rule(s) declared in .scalafix.conf on compilation and fail on lint errors," +
+          "with overlaying configuration by `triggered` sections if defined in .scalafix.conf file." +
```

Comment:
The plural in "sections" confused me a bit. I was using "overlay' as a verb in our previous discussions but I wonder if "overriden" isn't more clear in that context. What about:
```suggestion
        "Run Scalafix rule(s) declared in .scalafix.conf on compilation and fail on lint errors. " +
          "Default rules & rule configuration can be overriden using the `triggered` section." +
```